etc/build/doc

```diff
@@ -2,14 +2,10 @@
 
 set -e
 
-# we rename to pachctl because the program name is used in generating docs
-cp $GOPATH/bin/pachctl-doc ./pachctl
-
```

Comment:
Was this no longer needed. Or will this change the output of our docs?

Reply:
This is no longer needed - ` src/server/cmd/pachctl-doc/main.go` is now hard-coded to rewrite `os.Args[0]` to `pachctl` so it doesn't matter what the executable is named.